scripts/setup.sh

```
@@ -181,27 +197,39 @@ retry_until 60 5 'oc apply -f prerequisites/ca-issuer.yaml 2>/dev/null' || {
 
 # Apply authorino prerequisites and wait for it to be ready
-oc apply -f prerequisites/authorino-operator.yaml
-retry_until 300 3 '[[ -n "$(oc get csv --no-headers -n openshift-operators | grep authorino)" ]]' 'oc apply -f prerequisites/authorino-operator.yaml || true' || {
-    echo "Timed out waiting for authorino CSV to exist"
-    exit 1
-}
+if oc get deployment authorino-operator -n openshift-operators &>/dev/null; then
+    echo "Authorino operator is already installed, skipping..."
+else
+    oc apply -f prerequisites/authorino-operator.yaml
+    retry_until 300 3 '[[ -n "$(oc get csv --no-headers -n openshift-operators | grep authorino)" ]]' 'oc apply -f prerequisites/authorino-operator.yaml || true' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)